site/cds_rdm/clc_sync/services/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# CDS-RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CLCSync utils."""
+from flask import current_app
+import requests
+
+
+def clc_import(record_data):
+    """Import a record to CLC."""
+    clc_url = current_app.config.get("CLC_URL_SYNC")
+    clc_token = current_app.config.get("CDS_ILS_IMPORTER_API_KEY")
+    clc_payload = {
+        "data": record_data,
+        "mode": "IMPORT",
+    }
+
+    headers = {
+        "Content-Type": "application/vnd.inveniordm.v1+json",
+        "Accept": "application/vnd.inveniordm.v1+json",
```

Comment:
nit: we have the content type probably somewhere deep in a config var

Reply:
Hmm I see that we often add it in the service or resource level but not a general conf, unless I am mistaken.